Clarify usage definition (#220)
Qiskit/documentation: docs/run/estimate-job-run-time.mdx

# Estimate job run time

> After submitting a job to the IBM Quantum channel, you can see an estimation for how much _quantum time_ the job will take to run, by using `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.

Reviewer: ```suggestion
After submitting a job to the IBM Quantum channel, you can see an estimation for how much _quantum time_ the job will take to run by using `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.
```

> After submitting a job to the IBM Quantum channel, you can see an estimation for how much _quantum time_ the job will take to run. You can use `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.

Reviewer 2: Maybe it's just me, but this sentence

> You can use `job.usage_estimation`.

seems incomplete.

> After submitting a job to the IBM Quantum channel, you can see an estimation for how much _quantum time_ the job will take to run. You can use `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.

Reviewer: ```suggestion
After submitting a job to the IBM Quantum channel, you can see an estimation for how much _quantum time_ the job will take to run, by using `job.usage_estimation`.  Alternatively, you can view this information on the IBM Quantum Platform user interface.
```

> Quantum time is the duration a quantum system is committed to fulfilling a user request.

Reviewer 2: ```suggestion
Quantum time is the duration, in seconds, a quantum system is committed to fulfilling a user request.
```

```
## View the estimated job usage on IBM Quantum Platform

> You can view the estimated usage (in seconds) in two places on IBM Quantum Platform:

Reviewer 2: It's not at all obvious that "job usage" is the same as the quantum time that was mentioned above.

Author: Good point - I changed it to this "You can view the estimated usage (how much quantum time the job will take to run) in two places on IBM Quantum Platform"
